packages/e2e-tests/src/specs/editor/authorUser.js

```diff
@@ -75,9 +49,6 @@ describe('Author User', () => {
   //eslint-disable-next-line jest/no-disabled-tests
   it.skip('should be able to publish a story without markup being stripped', async () => {
     await createNewStory();
-    await expect(page).toMatchElement('.display-name', {
-      text: 'author',
-    });
```

Comment:
await expect(page).toMatch('Howdy, author');

Reply:
IIRC this didn't work in your testing, but happy to try again. Definitely a cleaner assertion 👍